dynawo/sources/Models/CPP/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ add_subdirectory(Common)
 add_subdirectory(ModelFrequency)
 add_subdirectory(Components/ModelLoadRestorativeWithLimits)
 add_subdirectory(Controls/Shunts/ModelCentralizedShuntsSectionControl)
+add_subdirectory(Controls/Voltage/ModelMinMaxMean)
```

Comment:
Maybe we could think about another name for this model ? I don't have a better proposal though...

Reply:
ModelMeasurementsUtiliies ? ModelVoltageMeasurementsUtilities ?